File: Ad-Hoc/__1000__/1893/1893.cpp
#include <iostream>

using namespace std;

int main() {

    int porcent1, porcent2;
    cin >> porcent1 >> porcent2;

    if ( porcent2 <= 2 && porcent2 >= 0 )
        cout << "nova" << endl;

    else if ( porcent2 <= 100 && porcent2 >= 97 )
        cout << "cheia" << endl;

    else if ( porcent1 < porcent2)
        cout << "crescente" << endl;

    else 
        cout << "minguante" << endl; 

    return 0;
}

File: Ad-Hoc/__1000__/1893/1893.rs
use std::io;

fn main() {

    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let data: Vec<u32> = input.trim().split_whitespace().map(|x| x.parse().unwrap()).collect();
    let (porcent2, porcent2) = (data[0], data[1]);

    if porcent2 <= 2 && porcent2 >= 0 { println!("nova");}

    else if porcent2 <= 100 && porcent2 >= 97 {  println!("cheia"); }

    else if porcent1 < porcent2 { println!("crescente"); } else { println!("minguante"); }
}